frameworks/person-escort-record/README.md

```diff
@@ -10,7 +10,7 @@ and [client applications](https://github.com/ministryofjustice/hmpps-book-secure
 
 A manifest is a list of sections in the PER defined as a tree-like structure. Each manifest file contains definitions of the steps and the questions that are required to be answered within each section.
 
-Manifest files should live in [`manifests/`](./manifests) and be stored in a flat structure.
+Manifest files should live in [`manifests/`](./manifests) and be stored in a flat structure. File names must use lower kebab casing, for example `file-name.yml`.
```

Comment:
@cwrw this was just what I've used for now but happy if we want to define this as camelCase to be safer across different environments?

Reply:
I'm happy with snake case as well for now 👍